examples/pom.xml

```diff
@@ -50,22 +51,26 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
+      <version>${hadoop.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>${hadoop.version}</version>
     </dependency>
-
-    <!-- Internal dependencies -->
     <dependency>
-      <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-client-fs</artifactId>
-      <version>${project.version}</version>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-hdfs</artifactId>
+      <version>${hadoop.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
     </dependency>
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-common</artifactId>
-      <version>${project.version}</version>
+      <artifactId>alluxio-shaded-client</artifactId>
+      <version>2.9.0</version>
```

Comment:
mvn will add this dependency and download it from maven repository.

Here we can also use "2.10.0-SNAPSHOT", for example, which needs to be built locally and installed from source code.

Reply:
why hardcode here?

Reply:
This example needs to be buildable independently. So it uses the `alluxio-shaded-client` as an external dependency.
The latest released `alluxio-shaded-client` is 2.9.0.

Our current version in code is 2.10.0-SNAPSHOT, but I need a way to compute the previous major release, e.g. 2.9.0.
Do we have a such way do so?

Reply:
Looks like no easy way.